CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.16.0] Q1 2024
+- added modem_info feature to make modemmanager a conditioanl dependency
+
```

Comment:
why not 0.15.1? imho this PR belongs to the modem feature which was 0.15.0

Reply:
IMHO it changes the way that the public interface (though at compile time) behaves in a downwards compatability breaking manner (thus one could even argue for a major version change, but as we are still on major version 0. this is not a biggy). The modem manager feature is not available for all configurations but must now explicitly be turned on.

We could probably merge the release w/ #73, though, if it is about too frequent minor version changes?

Reply:
from my point of view it is a fix of change for the 0.15.0 modem-manger feature. anyway - I'm fine with you explanation.